smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/CommandGenerator.java

```diff
@@ -344,7 +344,7 @@ private void generateEndpointParameterInstructionProvider() {
             operationContextParamValues.forEach((name, jmesPathForInputInJs) -> {
                 writer.write(
                     """
-                    $L: { type: \"operationContextParams\", name: $L },
+                    $L: { type: \"operationContextParams\", get: (input?: any) => $L },
```

Comment:
These need to be functions because there is no `this` at the time the instruction is evaluated. The instruction sits in the global scope due to the use of classBuilder to create commands.

```ts
export class XCommand extends $Command.
  classBuilder<...>()
  .ep({
    ...commonParams,
    MemberName: { type: "operationContextParams", get: (input?: any) => ... },
  })
  ...
  .build() {
  protected declare static __types: { ... }
}
```